roles/install/tasks/main.yml

```diff
@@ -0,0 +1,114 @@
+---
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2022, Felix Fontein
+
```

Comment:
What do you think about adding support for downloading sops's and age's published binaries and dropping them into /usr/local/bin? Some users maybe prefer this approach over debs and rpms, and this will allow installing age on distributions that don't have it packaged. That can be a later improvement, but I figured I'd suggest it :).

Reply:
I'm also wondering a bit whether we should really install GPG and Age, or only do that if requested by the user.

Reply:
I'm currently thinking of splitting this role up into three roles: one for installing sops, one for gpg, and one for age. Also installing age and gpg is mainly needed for the tests, and (for gpg) by users who actually want to use keys with these formats. Having options for this in the main role will mainly make the main role more complex, so I'd rather move that to separate roles.

Reply:
I've decided to split it up into two roles: `community.sops.install` installs sops and gpg (since sops uses the gpg executable in some cases), and the (for now) private role `community.sops._install_age` installs age. I've decided to keep `_install_age` private for now so we can figure out without pressure whether installing age should actually be part of this collection, and if yes, how exactly we want to support it (just from system packages, or also from sources, ...).